packages/ETLoad.package/ETLSegmentLoader.class/class/loadImageSegmentFrom.outPointers..st

```diff
@@ -17,4 +17,6 @@ loadImageSegmentFrom: segmentWordArray outPointers: outPointerArray
 		placeholder isLoadable ifFalse: [ ETLIncompatibleObjectError
 			signal: 'Loading classes is not yet supported'. ].
 		placeholder loadUsing: oopMap. ].
-	^ placeholders values first.
\ No newline at end of file
+	rootObject := placeholders values at: 1 ifAbsent: [ nil. ].
+	ETLLoadCallback value ifNotNilDo: [ :callback | callback cull: rootObject. ].
```

Comment:
What is the reason for using `cull` here? Shouldn't we expect the callback to always take exactly one argument?